text/000x-lowering-deposits-assethub.md

```diff
@@ -0,0 +1,197 @@
+
+This RFC suggests modifying deposit constants defined in the `nfts` pallet on the Polkadot Asset Hub to require a lower deposit. The reduced deposit amount should be determined by the `deposit` adjusted by the pricing mechanism (arbitrary number/another pricing function). 
+
+### Current code structure
+
+[Current deposit requirements are as follows](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L757)
+
+Looking at the current code structure the currently implemented we can find that the pricing re-uses the logic of how Uniques are defined:
+
+```rust
+parameter_types! {
+	// re-use the Uniques deposits
+	pub const NftsCollectionDeposit: Balance = UniquesCollectionDeposit::get();
+	pub const NftsItemDeposit: Balance = UniquesItemDeposit::get();
+	pub const NftsMetadataDepositBase: Balance = UniquesMetadataDepositBase::get();
+	pub const NftsAttributeDepositBase: Balance = UniquesAttributeDepositBase::get();
+	pub const NftsDepositPerByte: Balance = UniquesDepositPerByte::get();
+}
+```
+
+In the existing setup, the Uniques are defined with specific deposit values for different elements:
+
+```rust
+parameter_types! {
+	pub const UniquesCollectionDeposit: Balance = UNITS / 10; // 1 / 10 UNIT deposit to create a collection
+	pub const UniquesItemDeposit: Balance = UNITS / 1_000; // 1 / 1000 UNIT deposit to mint an item
+	pub const UniquesMetadataDepositBase: Balance = deposit(1, 129);
+	pub const UniquesAttributeDepositBase: Balance = deposit(1, 0);
+	pub const UniquesDepositPerByte: Balance = deposit(0, 1);
+}
+```
+As we can see in the code definition above the current code does not use the `deposit` funtion when the collection in the following instances: `UniquesCollectionDeposit` and  `UniquesItemDeposit`.
+
+### Proposed Modification Using the Deposit Function
+
+This proposed modification adjusts the deposits to use the `deposit` function instead of using an arbitrary number.
+
+```rust
+parameter_types! {
+	pub const NftsCollectionDeposit: Balance = deposit(1, 130);
+	pub const NftsItemDeposit: Balance = deposit(1, 164);
+	pub const NftsMetadataDepositBase: Balance = deposit(1, 129);
+	pub const NftsAttributeDepositBase: Balance = deposit(1, 0);
+	pub const NftsDepositPerByte: Balance = deposit(0, 1);
+}
+```
+
+Calculations viewed bellow were calculated by using the following repository [rfc-pricing](https://github.com/vikiival/rfc-pricing).
+**Polkadot**
+| **Name**                  | **Current price implementation** | **Proposed Modified by using the new deposit function** |
+|---------------------------|----------------------------------|-------------------------|
+| collectionDeposit         | 10 DOT                           | 0.20064 DOT             |
+| itemDeposit               | 0.01 DOT                         | 0.20081 DOT             |
```

Comment:
Interesting, 20x increase.

Reply:
That's why we propose a further reduction for _itemDeposit_ in the next paragraph by 40 
```rust 
pub const NftsItemDeposit: Balance = deposit(1, 164) / 40;
```